### ISSUE TITLE
Import multiple only loads one

### ISSUE THREAD

**Issue by Gubith** [Midnight-Lizard/Midnight-Lizard#324]
The import prompt shows imported several and to save each - but only one shows in the dropdowns.

**Comment by Gubith** [comment]
W10 20H2 (OS Build 19042.867)
Chrome Version 89.0.4389.128 (Official Build) (64-bit)

They do save properly one at a time.

**Comment by Gubith** [comment]
BTW - Midnight-Lizard is one of the most amazing works of Code in digital history.

**Comment by pavel-agarkov** [comment]
That's correct. You can find all imported color schemes in the drop-down menu but only one can be saves at a time.
